feature-libs/pickup-in-store/components/container/my-preferred-store/my-preferred-store.component.ts

```diff
@@ -31,11 +37,14 @@ export class MyPreferredStoreComponent implements OnInit {
   preferredStore$: Observable<PointOfService>;
   content = {
     header: 'My Store',
-    actions: [
-      { event: 'send', name: 'Get Directions' },
-      { event: 'edit', name: 'Change Store' },
-    ],
   };
+  actions = [
```

Comment:
I think this can be considered breaking too, let's try the `FeatureConfigService` here